scripts/live-connector-e2e/run-windows.ps1

```
@@ -0,0 +1,344 @@
+    [IO.Directory]::CreateDirectory($script:ToolRoot) | Out-Null
+    $package = if ($Connector -eq 'codex') { '@openai/codex@' + ($env:CODEX_VERSION ?? 'latest') } else { '@anthropic-ai/claude-code@' + ($env:CLAUDE_VERSION ?? 'latest') }
+    Invoke-Tool 'npm.cmd' @('install', '--no-audit', '--no-fund', '--prefix', $script:ToolRoot, $package) -Timeout 300 | Out-Null
+    $command = if ($Connector -eq 'codex') { 'codex.cmd' } else { 'claude.cmd' }
+    $script:AgentPath = Join-Path $script:ToolRoot "node_modules\.bin\$command"
+    $version = Invoke-NativeProcess -FilePath $script:AgentPath -ArgumentList @('--version') -TimeoutSeconds 30 -LogPath (Join-Path $script:LogRoot 'agent-version.log')
+    $script:AgentVersion = ($version.StdOut + $version.StdErr).Trim()
+    Write-Result install pass $script:AgentVersion
+}
+
+function Invoke-Agent([string]$Label, [string]$Prompt, [int[]]$AllowedExitCodes = @(0)) {
+    $args = if ($Connector -eq 'codex') {
```

PSScriptAnalyzer (1.25.0)

[warning] The Variable 'args' is an automatic variable that is built into PowerShell, assigning to it might have undesired side effects. If assignment is not by design, please use a different name.

(PSAvoidAssignmentToAutomaticVariable)